scaling.adoc

```diff
@@ -340,23 +338,23 @@ theme is that the participants create a number of semi-signed
 transactions that spend the output of the start transaction in
 different ways prior to publishing the start transaction. A
 semi-signed transaction can be made fully signed and put on the
-blockchain if someone misbehaves, to punish them. This keeps the
+blockchain if someone misbehaves to punish them. This keeps the
 participants' incentives aligned so that the protocol can work in a
 trustless way.
 
 After the start transaction is on the blockchain, the protocol can do
-what it's supposed to do, for example super-fast payment between
+what it's supposed to do, for example, super-fast payment between
 participants, or some privacy enhancing techniques, or to do more
 advanced scripting not supported on Bitcoin's blockchain.
 
-We won't go into more detail on how specific protocols work, but as
+We won't detail how specific protocols work, but as
 you can see in <<fig-scaling-layer>>, the blockchain is rarely used
 during the protocol's life cycle. All the juicy action happens
 _off-chain_. We've seen how this can be a win for <<privacy,privacy>>
 if done right, but it can also be a big win for scalability.
 
 In a https://www.reddit.com/r/Bitcoin/comments/438hx0/a_trip_to_the_moon_requires_a_rocket_with/[Reddit post] titled "`A trip to the moon requires a rocket with
-multiple stages or otherwise the rocket equation will eat your
+multiple stages or otherwise, the rocket equation will eat your
```

Comment:
This is a quote and should not be altered.

Reply:
thanks, addressed in https://github.com/kallerosenbaum/btcphilosophy/commit/0e8d06c4d600577181c6e1cd933bfc50db412809